src/main/java/frc/robot/subsystems/Hood.java

```diff
@@ -58,8 +60,17 @@ public boolean atSetpoint() {
    * @return the angle of the hood in degrees
    */
   public double getAngle() {
-    return (((m_pot.getVoltage() / Constants.AIO_MAX_VOLTAGE) * Constants.Hood.POT_SCALE) - Constants.Hood.POT_OFFSET)
-        * (Constants.Hood.MAX_ANGLE - Constants.Hood.MIN_ANGLE) + Constants.Hood.MIN_ANGLE;
+    return (getRawAngle() - m_bottomValue) * (Constants.Hood.MAX_ANGLE - Constants.Hood.MIN_ANGLE)
+        + Constants.Hood.MIN_ANGLE;
```

Comment:
Divide by raw expected range (Should be ~360deg)

Reply:
resolved in acedb0d